### DISCUSSION TITLE
Android app shows no issues when reporting feedback

### DISCUSSION THREAD

**Discussion by archon810** [community/community#31441] [Mobile]
I just reported a bug and I can't see it. Did it even go anywhere?

Because loading up the report feedback option shows no issues / reports, just an empty list, no matter what category is selected:

![Screenshot_20220830-211355.png](https://user-images.githubusercontent.com/469543/187591678-544b32d2-0ad1-4227-9929-be9579e38dff.png)

![Screenshot_20220830-211521.png](https://user-images.githubusercontent.com/469543/187591689-21df1363-36d6-42bd-8328-54f3ef9b2f6e.png)

Will this bug report get seen by anyone or also disappear into the void? Who knows...

**Comment by archon810** [comment]
Found my previous issue on the web version, so this seems like a bug in the Android app.

https://github.com/orgs/community/discussions/31440

**Comment by candyho** [comment]
Hey, we are seeing this report, don't worry. I'll check with our team to see what's going on here.

**Comment by candyho** [comment]
Do you mind sharing with us which mobile device are you using, your OS and the app version number? That'd definitely help us to speed up the investigation.

**Reply by archon810** [reply]
Pixel 6 Pro, Android 13 stable, app version  v1.78.0-beta (10082).

**Reply by jonyahmed09123** [reply]
Ok
